src/routes/edit.svelte

```diff
@@ -21,7 +21,23 @@
 		code: 'mermaid',
 		config: 'json'
 	};
+	const docMap = {
+		graph: '/#/flowchart',
+		graph_config: '/#/flowchart?id=configuration',
+		sequenceDiagram: '/#/sequenceDiagram',
+		sequenceDiagram_config: '/#/sequenceDiagram?id=configuration',
+		classDiagram: '/#/classDiagram',
+		classDiagram_config: '/#/classDiagram?id=configuration',
+		'stateDiagram-v2': '/#/stateDiagram',
+		gantt: '/#/gantt',
+		gantt_config: '/#/gantt?id=configuration',
+		pie: '/#/pie',
+		erDiagram: '/#/entityRelationshipDiagram',
+		erDiagram_config: '/#/entityRelationshipDiagram?id=styling',
+		journey: '/#/user-journey'
+	};
```

Comment:
```suggestion
	const docMap: <Appropriate type> = {
		graph: {
		    code: '/#/flowchart',
		    config: '/#/flowchart?id=configuration'
		},
		...
	};
```

Reply:
If config isn't available, there are 2 approaches I can think of

1. Link to code doc itself
2. Hide button

I think 1 might offer a better UX.

Reply:
agree with you. ^_^